jena-fuseki2/jena-fuseki-docker/download.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/sh
+
+## Licensed under the terms of http://www.apache.org/licenses/LICENSE-2.0
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ 
+# This is ash/dash script (it uses "local"), not a bash script.
```

Comment:
Missing space?

`ash/dash script`  -> `a sh/dash script`

Reply:
`ash` is the shell on Alpine. https://stackoverflow.com/questions/35689628/starting-a-shell-in-the-docker-alpine-container